util/util.lua

```diff
@@ -102,7 +102,7 @@ end
 copy = function ( dstpath, srcpath )
 
 	if osx then
-		print("COPY: \"" .. srcpath .. "\" -> \"" .. dstpath .. "\"")
+		--print("COPY: \"" .. srcpath .. "\" -> \"" .. dstpath .. "\"")
 		-- awful, but until there's a better way...
 		moaiCopy( srcpath, dstpath )
```

Comment:
I don't know why we need this moaiCopy - its probably redundant because ditto copies files with full attributes on osx.